apps/system/js/operator_variant_handler.js

```diff
@@ -279,7 +280,8 @@
           'ril.supl.passwd': 'password',
           'ril.supl.httpProxyHost': 'proxy',
           'ril.supl.httpProxyPort': 'port',
-          'ril.supl.authtype': 'authtype'
+          'ril.supl.authtype': 'authtype',
+          'ril.supl.mtu': 'mtu'
```

Comment:
missing in related test https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/operator_variant_handler_test.js#L68